lib/utils.js

```diff
@@ -30,7 +30,7 @@ function getFlagArrayFromObject(obj) {
       return obj[key] ? [...arr, '--no-update-notifier'] : arr;
     } else if (typeof obj[key] === 'boolean') {
       return obj[key] ? [...arr, flag] : arr;
-    } else if (key === 'ignoredFile') {
+    } else if (key === 'ignoredFile' || key === 'ignoredFiles') {
```

Comment:
Won't this break if ignoredFile is just a single string?

Reply:
Ok so after looking at how we document ignoring files in node-themekit, I realized I overlooked that ignoredFiles was an additional flag that we added on the wrapper. So I kept ignoredFiles to only support arrays, and ignored-file since its kebab case unlike most, I added a case for. I also removed ignoredFile allowing an array as its unintuitive especially when we have the additional ignoredFiles flag